src/objects/icon/icon.scss

```diff
@@ -0,0 +1,16 @@
+@use "sass:math";
+@use '../../mixins/svg';
+
+$inline-adjustment: 1rem / 16 * -2;
```

Comment:
It might be nice to include comments explaining this line and line 14.

Reply:
@Paul-Hebert I ended up re-writing this file to eliminate the need for embedding this math. I also added more comments to hopefully make it easier to understand. Let me know if this helped!

Reply:
Yep, much clearer and I think those reusable utilities will come in handy going forwards! :+1: